lib/babe/babe.go

```diff
@@ -142,6 +146,12 @@ func NewService(cfg *ServiceConfig) (*Service, error) {
 		return nil, err
 	}
 
+	metrics.Enabled = true
```

Comment:
should this be global? It would be uncommon for the node to run multiple `Service` instances, but may be useful for testing.

Reply:
I've setup this property only when `cfg.PublishMetrics` flags is true, for tests, it's possible to setup `metrics.Enable=true` since we don't have a lot of metrics tests